matchers/consist_of.go

```diff
@@ -16,35 +16,18 @@ type ConsistOfMatcher struct {
 	extraElements   []interface{}
 }
 
+var neighbours = func(v, m interface{}) (bool, error) {
```

Comment:
Maybe this should be defined as a function rather than a variable? This would make it constant so that you couldn't have bugs where it gets assigned to a different value.

Reply:
thanks :)